boost-maven/boost-maven-plugin/src/main/java/io/openliberty/boost/docker/dockerizer/DockerizeSpringBootClasspath.java

```diff
@@ -0,0 +1,240 @@
+    private static void extract(File artifact, File projectDirectory) {
+        File extractDir = new File(projectDirectory.getPath() + "/target/dependency/");
+        if (!extractDir.exists())
+            extractDir.mkdirs();
+
+        try {
+            byte[] buffer = new byte[1024];
+            ZipInputStream zis = new ZipInputStream(new FileInputStream(artifact));
+            ZipEntry ze = zis.getNextEntry();
+            while (ze != null) {
+                File newFile = new File(extractDir, ze.getName());
+                if (ze.isDirectory()) {
+                    newFile.mkdirs();
+                } else {
+                    new File(newFile.getParent()).mkdirs();
+                    FileOutputStream fos = new FileOutputStream(newFile);
+                    int len;
+                    while ((len = zis.read(buffer)) > 0) {
+                        fos.write(buffer, 0, len);
+                    }
+                    fos.close();
+                }
+                ze = zis.getNextEntry();
+            }
+            // close last ZipEntry
+            zis.closeEntry();
+            zis.close();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+    }
+
+    private String getSpringStartClass(File artifact) {
+        try (JarFile jarFile = new JarFile(artifact)) {
+            Manifest manifest = jarFile.getManifest();
+            if (manifest != null) {
+                Attributes attributes = manifest.getMainAttributes();
+                return attributes.getValue("Start-Class");
+            }
+        } catch (IOException e) {}
+        return "";
+    }
+
+    private File createNewDockerFile(File dockerfileDirectory) throws IOException {
+        File dockerFile = new File(dockerfileDirectory, "Dockerfile");
+        Files.createFile(dockerFile.toPath());
+        log.info("Creating Dockerfile: " + dockerFile.getAbsolutePath());
+        return dockerFile;
+    }
+
+    private void writeSpringBootDockerFile(File dockerFile, String startClass) throws IOException {
+        ArrayList<String> lines = new ArrayList<>();
+        lines.add(BOOST_GEN);
+        lines.add("FROM adoptopenjdk/openjdk8-openj9");
+        lines.add("VOLUME /tmp");
+        lines.add("ARG DEPENDENCY=target/dependency");
+        lines.add("COPY ${DEPENDENCY}/BOOT-INF/lib /app/lib");
+        lines.add("COPY ${DEPENDENCY}/META-INF /app/META-INF");
+        lines.add("COPY ${DEPENDENCY}/BOOT-INF/classes /app");
+        lines.add("ENTRYPOINT [\"java\",\"-cp\",\"app:app/lib/*\",\"" + startClass + "\"]");
+        Files.write(dockerFile.toPath(), lines, Charset.forName("UTF-8"));
+    }
+
+    /**
+     * Creates a new .dockerignore file in the project directory if the
+     * .dockerignore file doesn't exist. Existing .dockerignore file will be
+     * appended with the lines generated by the plugin. Creating/appending of
+     * .dockerignore happens only when the plugin generates Dockerfile.
```

Comment:
Are you sure about this? Based on what you're writing to the .dockerignore file (namely `target/liberty`), that seems like something we would always want to ignore as long as they're using Boost.